### ISSUE TITLE
Better upload_uri support for bin/cpan-upload

### ISSUE THREAD

**Issue by tyldum** [rjbs/CPAN-Uploader#24]
So setting upload_uri in the config file did not work for me, and looking at the code I can't find any reason it should.
Digging through the code I did find that I could set the upload_uri by using $ENV{CPAN_UPLOADER_UPLOAD_URI}.

Before submitting a PR, what is the preferred approach for the current maintainers?

1. Update documentation to mention CPAN_UPLOADER_UPLOAD_URI?
2. Add support for a command line option --upload_uri?
3. Add support for reading it from the config file? (it is currently read, but not used)

Would come in really handy as I'm perfecting our CI/CD pipeline with an upload to App::opan.
I prefer to do 2. and 3., and can happily send a PR for those.

**Comment by srchulo** [comment]
Yeah, I would really like an `--upload_uri` option for `cpan-upload`. Currently I'm using `$ENV{CPAN_UPLOADER_UPLOAD_URI}` instead.

I'm not sure about using the `.pause` file directly with `CPAN::Uploader` not working, but when I use `Minilla` my `.pause` file with `upload_url` does work.
